src/internal/testutil/directory.go

```diff
@@ -0,0 +1,27 @@
+package testutil
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+)
+
+// MkdirTemp creates a temporary directory with a random name under the OS's
+// temp directory and deletes it at the end of the test.
+func MkdirTemp(t testing.TB) string {
```

Comment:
This is provided by the testing package.
https://golang.org/pkg/testing/#T.TempDir

Reply:
That seems useful, I'll move over to that.